apps/andi/lib/andi_web/views/error_helpers.ex

```diff
@@ -89,9 +89,8 @@ defmodule AndiWeb.ErrorHelpers do
 
   defp interpret_error_message(_message, :schema, _), do: "Please add a field to continue"
 
-  defp interpret_error_message(_message, :datasetLink, _), do: "Please enter a valid URL to your externally-hosted dataset"
```

Comment:
I added this error message as a part of a card that gave the user more descriptive error messages. I would check with Jake to see if we want a more descriptive message than the default

Reply:
The error messaging is all done in the event handlers for this field. Essentially for the size being to large, the format, or an error uploading the file